### PR TITLE
Limit id min-max width on product list

### LINKED ISSUE #24915: ID column is too large
#### Describe the bug

Hello,

I noticed not long ago that the ID column has been changed even though I haven't touched anything. Do you know why?

Thanks

**Screenshots**

image

#### Additional information

* PrestaShop version: 1.7.6 and 1.7.7
* PHP version: 5.6

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | The ID head column was too big on product list
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #24915.
| How to test?      | Go on product list and see the id head width should me smaller as before
| Possible impacts? | ID head column width

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24952)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss [hunk 1]
```diff
@@ -57,6 +57,10 @@
       font-weight: 400;
     }
 
+    #filter_column_id_product_div {
```
Comment (kpodemski) [inline admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: @NeOMakinG from what I know `id` column is too big on every page, Categories too, I think we need something more generic?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: Ok, gonna think about it

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: @kpodemski would you mind giving help on this? It seems that you're right, some id's are big too, but I've no way of selecting them globally as they doesn't have a specific class everywhere :/

I would expect something like a class named `grid-head-id` on every head related to the id column

Comment (kpodemski) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: @NeOMakinG 

https://github.com/PrestaShop/PrestaShop/issues/22847

haha :) so now we have use case, I'll handle this tomorrow 👍

Comment (kpodemski) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: @NeOMakinG done, you can check if that's ok :)

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: Ok, gonna look at it today :)

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/product/products_catalog.scss:60]: Thanks!

**Review by jolelievre — COMMENTED** [review]
I believe the bug is present on 178x as well, so it should target 178x branch as it looks like a regression

**Comment by NeOMakinG** [comment]
> I believe the bug is present on 178x as well, so it should target 178x branch as it looks like a regression

1.7.6, 1.7.7, 1.7.8 and develop, looks like the bug is here since years, so I doubt we can consider it as a regression, wdyt?

**Comment by kpodemski** [comment]
@NeOMakinG 

problem is that for years it wasn't visible, recent Chrome update did something and now it's entirely broken 😱

**Comment by NeOMakinG** [comment]
> @NeOMakinG
> 
> problem is that for years it wasn't visible, recent Chrome update did something and now it's entirely broken 😱

Did we prove this? I remember seeing this since a lot of time, but maybe I'm wrong

As you want anyway, I can cherry-pick these commit on 1.7.8 (as I always fails the develop to 1.7.8 rebase)

**Comment by NeOMakinG** [comment]
It's more global now @kpodemski, fixed the name field which had the same problem on some grids

**Comment by kpodemski** [comment]
@NeOMakinG LGTM, the only edge scenario I can think of, is when someone has an identifier which is something like `entity_id`, not `id_entity` 🤔

**Comment by NeOMakinG** [comment]
> @NeOMakinG LGTM, the only edge scenario I can think of, is when someone has an identifier which is something like `entity_id`, not `id_entity` 🤔

This is something possible, or not even containing id but the `identifier` keyword, but if we find this edge case, we'll adapt it

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
@NeOMakinG Always in WIP ?

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @sarahdib

**Pull request merged**